jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/impl/PostgreSQLSqlDialect.java

```diff
@@ -0,0 +1,404 @@
+
+        // Conversion functions
+//        cap.supportScalarFunction(ScalarFunctionCapability.IS_NUMBER);
+//        cap.supportScalarFunction(ScalarFunctionCapability.IS_BOOLEAN);
+//        cap.supportScalarFunction(ScalarFunctionCapability.IS_DATE);
+//        cap.supportScalarFunction(ScalarFunctionCapability.IS_DSINTERVAL);
+//        cap.supportScalarFunction(ScalarFunctionCapability.IS_YMINTERVAL);
+//        cap.supportScalarFunction(ScalarFunctionCapability.IS_TIMESTAMP);
+        cap.supportScalarFunction(ScalarFunctionCapability.TO_CHAR);
+//        cap.supportScalarFunction(ScalarFunctionCapability.TO_DATE);
+//        cap.supportScalarFunction(ScalarFunctionCapability.TO_DSINTERVAL);
+//        cap.supportScalarFunction(ScalarFunctionCapability.TO_YMINTERVAL);
+//        cap.supportScalarFunction(ScalarFunctionCapability.TO_NUMBER);
+//        cap.supportScalarFunction(ScalarFunctionCapability.TO_TIMESTAMP);
+        
+        // Bitwise functions
+//        cap.supportScalarFunction(ScalarFunctionCapability.BIT_AND);
+//        cap.supportScalarFunction(ScalarFunctionCapability.BIT_CHECK);
+//        cap.supportScalarFunction(ScalarFunctionCapability.BIT_NOT);
+//        cap.supportScalarFunction(ScalarFunctionCapability.BIT_OR);
+//        cap.supportScalarFunction(ScalarFunctionCapability.BIT_SET);
+//        cap.supportScalarFunction(ScalarFunctionCapability.BIT_TO_NUM);
+//        cap.supportScalarFunction(ScalarFunctionCapability.BIT_XOR);
+        
+        
+     // Other functions
+        cap.supportScalarFunction(ScalarFunctionCapability.CASE);
+//        cap.supportScalarFunction(ScalarFunctionCapability.CURRENT_SCHEMA);
+//        cap.supportScalarFunction(ScalarFunctionCapability.CURRENT_SESSION);
+//        cap.supportScalarFunction(ScalarFunctionCapability.CURRENT_STATEMENT);
+//        cap.supportScalarFunction(ScalarFunctionCapability.CURRENT_USER);
+        cap.supportScalarFunction(ScalarFunctionCapability.HASH_MD5);
+//        cap.supportScalarFunction(ScalarFunctionCapability.HASH_SHA);
+//        cap.supportScalarFunction(ScalarFunctionCapability.HASH_SHA1);
+//        cap.supportScalarFunction(ScalarFunctionCapability.HASH_TIGER);
+//        cap.supportScalarFunction(ScalarFunctionCapability.NULLIFZERO);
+//        cap.supportScalarFunction(ScalarFunctionCapability.SYS_GUID);
+//        cap.supportScalarFunction(ScalarFunctionCapability.ZEROIFNULL);
+        
+        return cap;
+	}
+	
+	 @Override
+    public DataType mapJdbcType(ResultSet cols) throws SQLException {
+        DataType colType = null;
+        int jdbcType = cols.getInt("DATA_TYPE");
+
+        switch (jdbcType) {
+        		case Types.OTHER:
+	            	String columnTypeName = cols.getString("TYPE_NAME");
+	
+	            	if(columnTypeName.equals("varbit")){
+	            			int n = cols.getInt("COLUMN_SIZE");
+	            		
+	           		 		colType = DataType.createVarChar(n, DataType.ExaCharset.UTF8);
```

Comment:
This indentation looks strange, probably mix of tabs/space. Probably makes sense to add checkstyle plugin to pom, with just a single rule that there may not be any tabs.